src/ontology/mappings.tsv

```diff
@@ -166,23 +165,20 @@ FBbt:00004505	UBERON:0003161	exact	ocellus
 FBbt:00004506	UBERON:0003162	exact	lateral ocellus
 FBbt:00004751	UBERON:0003194	exact	wing vein
 FBbt:00004894	UBERON:0003199	exact	egg chamber	Uberon term commented as “candidate for obsoletion”
-FBbt:00004973	UBERON:0003201	exact	exocuticle	FBbt’s definition makes it pupa-specific
+FBbt:00004973	UBERON:0003201	exact	exocuticle
 FBbt:00004974	UBERON:0003202	exact	endocuticle	Conflicting definition (inner layer in Uberon; middle layer in FBbt)
 FBbt:00004507	UBERON:0003211	exact	medial ocellus	Uberon term should be under ‘dorsal ocellus’ (CL:0003161)
 FBbt:00005159	UBERON:0003212	exact	gustatory sensory organ
 FBbt:00007474	UBERON:0003914	exact	epithelial tube
 FBbt:00005066	UBERON:0003917	exact	fat body	Uberon definition makes it a single gland localized dorsally to the gut; FBbt definition makes it distributed throughout the body; and not sure it's correct to define it as a gland
 FBbt:00025998	UBERON:0004120	exact	mesodermal derivative
 FBbt:00025990	UBERON:0004121	exact	ectodermal derivative
-FBbt:00005140	UBERON:0004732	exact	neuromere	Should point to neuromere (Uberon:0004731)?
+FBbt:00005140	UBERON:0004731	exact	neuromere
 FBbt:00005317	UBERON:0004734	exact	gastrula embryo	marked as deprecated in FBbt (but without suggested replacement)
 FBbt:00001956	UBERON:0004904	exact	optic nerve	To check
 FBbt:00005811	UBERON:0004905	exact	articulation
-FBbt:00004859	UBERON:0004909	exact	gonadal sheath	Source term should be ‘gonadal sheath epithelium’ (FBbt:00047208)
-FBbt:00004955	UBERON:0004910	exact	testis sheath	The FBbt term does not refer to an epithelium (the testis sheath contains an epithelial layer but also a muscular layer); maybe FBbt should have a 'male gonadal sheath epithelium'?
-FBbt:00004896	UBERON:0004911	exact	ovarian sheath	Likewise
+FBbt:00047208	UBERON:0004909	exact	gonadal sheath epithelium
```

Comment:
Equivalence may be too strong here. Note also Uberon term looks dodgy to me.  Its text def says epithelium surrounding a gonad.  Its logical def covers all epithelia of gonad.  Not clear that all are on surface (see granulosa cell layer) and anyway there are bound to be species where this doesn't apply.

Reply:
Likewise.